source/index.txt

```diff
@@ -1,80 +1,260 @@
+
+         .. step:: Run Aggregation Pipelines
+
+            You can run aggregation pipelines in :binary:`~bin.mongosh` 
+            using the :method:`db.collection.aggregate()` method. 
+            Aggregation pipelines transform your documents into 
+            aggregated results based on the stages you specify. For more 
+            information and examples, refer to :ref:`mdb-shell-aggregation`.
+
+      .. image:: /images/mongosh-plp/Technical_ACTION_SearchDocument_Thumbnail_BS_Mist.png
+         :alt: Abstract illustration showing CRUD and aggregation operations
+
+   .. tab:: Administer and Manage
+      :tabid: administer
+
+      .. procedure::
+
+         .. step:: Manage Databases and Collections
+
+            View information about databases, create collections or views,
+            or drop databases - all from your shell. See all 
+            :ref:`Database Methods <mongosh-database-methods>`.
+
+            Perform collection operations, create or delete indexes, or 
+            explain queries with :ref:`Collection Methods 
+            <mongosh-collection-methods>`.
+
+         .. step:: Administer Servers
+
+            Manage :ref:`replication <mongosh-replication-methods>` or 
+            :ref:`sharding <mongosh-sharding-methods>` conveniently in 
+            your shell.
+
+            Check server status with a variety of :ref:`Server Status Methods 
+            <mongosh-server-status-methods>`.
+
+         .. step:: Manage Users and Roles
+
+            Create or update roles, define and update privileges, or drop 
+            roles using :ref:`Role Management Methods 
+            <mongosh-role-management-methods>`.
+
+            Create and update users, authenticate users, and manage user
+            roles with :ref:`User Management Methods 
+            <mongosh-user-management-methods>`. 
+
+      .. image:: /images/mongosh-plp/Technical_SOFTWARE_Terminal(1)_Spot.png
+         :alt: Abstract illustration showing a terminal performing operations
+
+   .. tab:: Automate with Scripts
+      :tabid: scripts
+
+      .. procedure::
+
+         .. step:: Run Scripts for CRUD or Administrative Tasks
+
+            Write scripts for the |mdb-shell| that perform CRUD or 
+            administrative operations in MongoDB.
+
+            For example, if you have a JS file that seeds synthetic or
+            mock data into MongoDB in your development or staging 
+            environment, run the file with:
+
+            .. code-block:: sh
+
+               mongosh YOUR_JS_FILENAME.js
+
+            Explore a tutorial that uses the |mdb-shell| with JavaScript to
+            access MongoDB: :ref:`mdb-shell-write-scripts`.
 
-.. default-domain:: mongodb
+         .. step:: Create Custom Helper Functions with .mongoshrc
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+            If you often find yourself converting date strings to ISO format
```

Comment:
Suggestion:

Repeatedly writing large helper functions in the Shell? Store them in a [.mongoshrc config file](https://preview-mongodbdacharyc.gatsbyjs.io/mongodb-shell/DOCSP-32028/mongoshrc/#std-label-mongoshrc-js). For example, if you often find yourself converting date strings to ISO format for queries, create a function in .mongoshrc to handle it: ...